README.md

```diff
@@ -81,9 +81,9 @@ needed.
 To be able to create an account on your test environment an SMTP server is
 required. A simple debugging SMTP server can be setup using Python.
 
-        python2 -m smtpd -n -c DebuggingServer localhost:1025
+        python -m smtpd -n -c DebuggingServer localhost:1025
 
-In local_settings.py change the EMAIL_HOST to 'localhost' and the EMAIL_PORT to
+In local\_settings.py add entries to set EMAIL\_HOST to 'localhost' and EMAIL\_PORT to
```

Comment:
I don't see why this is required? It renders fine on Github.

Reply:
local_settings.py doesn't have existing values for EMAIL_HOST and EMAIL_PORT, so I changed the wording to say "add" vs. "change".

Reply:
Yeah but note the escaping of _ which wasn't there ;-)

Reply:
Ah - Vim was showing it as a red parsing error without the escape, and it's escaped in other places, so I added them. I'm no markdown expert or purist, so I can take them out again, if you like?

Reply:
I think putting local_settings.py, email_host and email_port into a code block would probably be better